prime-router/src/test/kotlin/ElementTests.kt

```diff
@@ -17,6 +17,8 @@ import java.time.format.DateTimeFormatter
 import kotlin.test.Test
 import kotlin.test.assertFails
 
+private const val yyyyMMddStrg = "yyyy-MM-dd"
```

Comment:
We don't use Hungarian notation for variable names, so including the fact that this is a string in the name runs counter to best practices. It's better to name the variable after it's intent, something like `dateFormat`. Please change it.

Reply:
Thank you, I am changing it.